flexmeasures/ui/crud/assets.py

```diff
@@ -321,8 +325,31 @@ def status(self, id: str):
         asset = process_internal_api_response(asset_dict, int(id), make_obj=True)
         status_data = build_sensor_status_data(asset)
 
+        # add data abount max_jobs_num forecasting and scheduling jobs
+        redis_connection_err = None
+        scheduling_job_data, forecasting_job_data = list(), list()
+        try:
+            jobs_data = build_asset_jobs_data(asset)
+        except NoRedisConfigured as e:
+            redis_connection_err = e.args[0]
+        else:
+            max_jobs_num = 10
```

Comment:
Thought: If jobs disappear by themselves, we might not need `max_jobs_num` - the list would be as ling as TTL governs. It even might hide an important 11th job from us.